catalog/transformer/catalog-transformer-tika-input/src/test/java/ddf/catalog/transformer/input/tika/TikaInputTransformerTest.java

```diff
@@ -81,17 +82,17 @@
 
   private static final String COMMON_METACARDTYPE_NAME = "fallback.common";
 
-  private static final String SOUND = "Sound";
+  private static final String SOUND = DataType.SOUND.toString();
 
-  private static final String IMAGE = "Image";
+  private static final String IMAGE = DataType.IMAGE.toString();
 
-  private static final String TEXT = "Text";
+  private static final String TEXT = DataType.TEXT.toString();
 
-  private static final String DOCUMENT = "Document";
+  private static final String DOCUMENT = DataType.TEXT.toString();
```

Comment:
❓ Should we get rid of `DOCUMENT` if it doesn't apply anymore?

Reply:
I did wonder about that. I can remove it and use the text one everywhere if you think we should do that.